openshift-kubecost-install.md

```diff
@@ -0,0 +1,513 @@
+Install Kubecost on Redhat Openshift cluster
+==================
+
+## Overview
+
+This documentation provides instructions on how to deploy Kubecost on Redhat Openshift cluster. Additional security settings may also be required. You can refer to the Architecture overview and installation section for more information. 
+
+---
+## Architecture overview:
+
+Currently there are 2 main options to deploy Kubecost on Redhat Openshift Cluster (OCP).
+
+### Standard deployment:
+
+Kubecost is installed with Cost Analyzer and Prometheus as a time-series database. Data is gathered by the Prometheus installed with Kubecost (user-defined Prometheus). Other metrics are scraped by user-defined Prometheus from OCP monitoring stack managed components like KSM, OSM, CAdvisor, etc .... Kubecost then push and query metrics to/from user-defined Prometheus. Enterprise set up could also work with Thanos as additional component.
```

Comment:
Why are we saying "user-defined Prometheus" if it's bundled with our helm chart?

Reply:
I used the language from OpenShift doc actually. Nice catch, I will edit it in the next update for entire doc.

Reply:
It is updated in the latest commit.